src/tests/jsonLogicFixtures.js

```diff
@@ -0,0 +1,1098 @@
+export const schemaWithComputedAttributes = {
+  properties: {
+    field_a: {
+      type: 'number',
+    },
+    field_b: {
+      type: 'number',
+      'x-jsf-computedAttributes': {
+        title: 'This is {{a_times_two}}!',
+        value: 'a_times_two',
+        description: 'This field is 2 times bigger than field_a with value of {{a_times_two}}.',
+      },
+    },
+  },
+  required: ['field_a', 'field_b'],
+  'x-jsf-logic': {
+    computedValues: {
+      a_times_two: {
+        rule: {
+          '*': [{ var: 'field_a' }, 2],
+        },
+      },
+    },
+  },
+};
+
+export const nestedFieldsetWithValidationSchema = {
+  properties: {
+    field_a: {
+      type: 'object',
+      'x-jsf-presentation': {
+        inputType: 'fieldset',
+      },
+      properties: {
+        child: {
+          type: 'number',
+          'x-jsf-requiredValidations': ['child_greater_than_10'],
+        },
+      },
+      required: ['child'],
+      'x-jsf-logic': {
+        validations: {
+          child_greater_than_10: {
+            errorMessage: 'Must be greater than 10!',
+            rule: {
+              '>': [{ var: 'child' }, 10],
+            },
+          },
+        },
+      },
+    },
+  },
+  required: ['field_a'],
+};
+
+export const validatingTwoNestedFieldsSchema = {
+  properties: {
+    field_a: {
+      type: 'object',
+      'x-jsf-presentation': {
+        inputType: 'fieldset',
+      },
+      properties: {
+        child: {
+          type: 'number',
+          'x-jsf-requiredValidations': ['child_greater_than_10'],
+        },
+        other_child: {
+          type: 'number',
+          'x-jsf-requiredValidations': ['greater_than_child'],
+        },
+      },
+      required: ['child', 'other_child'],
+      'x-jsf-logic': {
+        validations: {
+          child_greater_than_10: {
+            errorMessage: 'Must be greater than 10!',
+            rule: {
+              '>': [{ var: 'child' }, 10],
+            },
+          },
+          greater_than_child: {
+            errorMessage: 'Must be greater than child',
+            rule: {
+              '>': [{ var: 'other_child' }, { var: 'child' }],
+            },
+          },
+        },
+      },
+    },
+  },
+  required: ['field_a'],
+};
+
+export const twoLevelsOfJSONLogicSchema = {
+  properties: {
+    field_a: {
+      type: 'object',
+      'x-jsf-presentation': {
+        inputType: 'fieldset',
+      },
+      properties: {
+        child: {
+          type: 'number',
+          'x-jsf-requiredValidations': ['child_greater_than_10'],
+        },
+      },
+      required: ['child'],
+      'x-jsf-logic': {
+        validations: {
+          child_greater_than_10: {
+            errorMessage: 'Must be greater than 10!',
+            rule: {
+              '>': [{ var: 'child' }, 10],
+            },
+          },
+        },
+      },
+    },
+    field_b: {
+      type: 'number',
+      'x-jsf-requiredValidations': ['validation_parent', 'peek_to_nested'],
```

Comment:
question: Why is the `peek_to_nested` here if it's about `field_a.child` which has no influence under `field_b`?

It makes me wonder if a validation at `requiredValidations` MUST always have something in common with the field itself.